src/wp-includes/block-supports/dimensions.php

```diff
@@ -44,23 +43,39 @@ function wp_register_dimensions_support( $block_type ) {
  * This will be applied to the block markup in the front-end.
  *
  * @since 5.9.0
+ * @since 6.2.0 Added `minHeight` support.
  * @access private
  *
  * @param WP_Block_Type $block_type       Block Type.
  * @param array         $block_attributes Block attributes.
  * @return array Block dimensions CSS classes and inline styles.
  */
 function wp_apply_dimensions_support( $block_type, $block_attributes ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
-	if ( wp_should_skip_block_supports_serialization( $block_type, '__experimentalDimensions' ) ) {
+	if ( wp_should_skip_block_supports_serialization( $block_type, 'dimensions' ) ) {
```

Comment:
Same BC discussion here. See https://github.com/WordPress/wordpress-develop/pull/3940/files#r1090737459 and https://github.com/WordPress/wordpress-develop/pull/3940/files#r1090745794.

Reply:
tl;dr There's no BC concern as it was never wired to anything before 6.2. See explanation https://github.com/WordPress/wordpress-develop/pull/3940#discussion_r1092546821.